src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -46,7 +46,7 @@ public ZipkinClient(ILog logger, IZipkinConfig zipkinConfig, IOwinContext contex
                     spanCollector,
                     new ServiceEndpoint(),
                     zipkinConfig.NotToBeDisplayedDomainList,
-                    zipkinConfig.Domain);
+                    zipkinConfig.Domain(context.Request));
```

Comment:
Do we need a `Func` for a configuration setting? What are the drawbacks of just having something like:

```C#
zipkinConfig.Domain ?? context.Request.Uri.Host
```